conf/modules.config

```diff
@@ -52,6 +52,55 @@ process {
         ext.prefix = { "${meta.id}.fa" }
     }
 
+    withName: MMSEQS_DATABASES {
+        publishDir = [
+            path: { "${params.outdir}/databases/" },
```

Comment:
```suggestion
            path: { "${params.outdir}/databases/mmseqs/" },
```

Reply:
it saves it to a mmseqs_taxonomy directory by default, thats why i didnt want to add mmseqs here, because then it will save to `mmseqs/mmseqs_taxonomy/...`

Reply:
Will there be other such folders from mmseqs (also in future)? Instead of

- `.../databases/mmseqstaxonomy/`
- `.../databases/mmseqsmoredatabase/`
- `.../databases/mmseqsblubb/`

I would rather have all that in a `.../databases/mmseqs/` subfolder. If no further mmseqs dbs are expected, go for your option :+1: